### PR TITLE
KNL-1541 - Missing fields initializer for change

### PR DESCRIPTION
**Pull request by jonespm**

### CODE REVIEW

File: kernel/kernel-impl/src/main/java/org/sakaiproject/site/impl/DbSiteService.java [hunk 1]
```diff
@@ -1999,6 +1999,7 @@ else if (permission == 1)
 				batchList = new ArrayList<>();
 				for (String siteId : visitInserts)
 				{
+					fields = new Object[3];
 					fields[0] = caseId(siteId);
 					fields[1] = userId;
 					fields[2] = Integer.valueOf(1);
```
Comment (buckett) [inline kernel/kernel-impl/src/main/java/org/sakaiproject/site/impl/DbSiteService.java]: Shouldn't the adding of the `fields` to the `batchList` also be here?

Comment (jonespm) [reply kernel/kernel-impl/src/main/java/org/sakaiproject/site/impl/DbSiteService.java]: Yeah looks like there's a few bugs in here. thanks.

Comment (jonespm) [reply kernel/kernel-impl/src/main/java/org/sakaiproject/site/impl/DbSiteService.java]: Updated PR, looks like it probably only needs to do the write at the end of generating all these batches too.

Comment (ottenhoff) [reply kernel/kernel-impl/src/main/java/org/sakaiproject/site/impl/DbSiteService.java]: yeah, i missed that these were all the same PreparedStatement. Your change LGTM

**Pull request merged**